handlers/ticket-tailor-webhook/src/index.ts

```diff
@@ -24,21 +24,26 @@ export const handler: Handler = async (event: SQSEvent) => {
 			currentDateTime,
 		);
 		if (!validRequest) {
-			throw new Error(
-				'Signatures do not match - check Ticket Tailor signing secret matches the one stored in AWS.',
-			);
+			console.error('Request failed validation. Processing terminated.');
```

Comment:
I'm not 100% sure on this log and continue style, it does feel more functional and presumably we deicded we want the item to still be removed from the queue?  Rather than building up in the DLQ?  I'm not sure I 100% agree, we would want to know asap if the signatures are not being validated generally?
If so pull out the "else" into a separate function to make it clear that everything after the else will run regardless.  Otherwise if someone later adds some logic afterwards, it would happen even on an invalid signature..
```suggestion
			console.error('Request failed validation. Skipping over with "success" to remove it from the queue.');
```

Reply:
Resolved this in person and implemented changes to this effect.